apps/Gruntfile.js

```diff
@@ -542,6 +542,7 @@ describe('entry tests', () => {
     'levels/_bubble_choice':
       './src/sites/studio/pages/levels/_bubble_choice.js',
     'levels/_content': './src/sites/studio/pages/levels/_content.js',
+    'levels/_teacher_markdown': './src/sites/studio/pages/levels/_teacher_markdown.js',
```

Comment:
I hate to make style comments now that we have prettier, but can you please preserve the alphabetical order?